updater/Windows/WindowsUpdater.cpp

```diff
@@ -445,8 +445,8 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPWSTR lpCmdLi
 	if (argc != 5)
 	{
 		progress.ModalError("Expected 4 arguments: parent process id, output directory, update zip, program to "
-							"launch.\n\nThis program is not intended to be run manually, please use the Qt frontend and "
-							"click Help->Check for Updates.");
+							"launch.\n\nThis program is not intended to be run manually, please use the main PCSX2 application and "
+							"click Help -> Check for Updates.");
```

Comment:
I don't think the spaces either side are required here.

Reply:
Thats just my OCD getting in the way again xD
Anyway i'll change that back